compiler/aarch64/codegen/OpBinary.cpp

```diff
@@ -476,5 +476,12 @@ const OMR::ARM64::InstOpCode::OpCodeBinaryEntry OMR::ARM64::InstOpCode::binaryEn
 		0x1E604800,	/* FMAX      	fmaxd	 */
 		0x1E205800,	/* FMIN      	fmins	 */
 		0x1E605800,	/* FMIN      	fmind	 */
+	/* Vector Data-processing (2 source) */
+		0x4EA01C00,	/* ORR      	vorr2d	 */
+		0x4E208400,	/* ADD      	vadd16b	 */
+		0x4E608400,	/* ADD      	vadd8h	 */
+		0x4E20D400,	/* FADD      	vfadd4s	 */
+		0x4E60D400,	/* FADD      	vfadd2d	 */
+/* Hint instructions */
```

Comment:
Fix indentation to be consistent.

Reply:
This indention is aligned with `/* VFP instructions */`, would you please confirm if I need to change this? Thanks.

Reply:
`/* Hint instructions */` is not aligned in the same way as similarly-themed comments above it.